SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/LayeredLBStrategy/TestLayeredLBStrategy.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include "CoreMinimal.h"
+#include "LoadBalancing/GridBasedLBStrategy.h"
+#include "GameFramework/DefaultPawn.h"
+#include "TestLayeredLBStrategy.generated.h"
+
+/**
+ * This class is for testing purposes only.
+ */
+UCLASS(HideDropdown)
+class SPATIALGDKTESTS_API UTwoByFourLBGridStrategy : public UGridBasedLBStrategy {
+	GENERATED_BODY()
+
+public:
+	UTwoByFourLBGridStrategy(): Super() {
+		Rows = 2;
+		Cols = 4;
+	}
+};
+
+/**
+ * Same as a Default pawn but for testing
+ */
+UCLASS(config = Game, Blueprintable, BlueprintType)
+class SPATIALGDKTESTS_API ASpecialPawn : public ADefaultPawn
```

Comment:
Reading through this, I found the use of DefaultPawn and SpecialPawn a bit confusing. I'd suggest having Layer1Pawn and Layer2Pawn, then specifically adding them to those layers and naming them things like Layer1Actor. Then only use the DefaultPawn if you are testing for something being assigned to the default layer. It just makes the more complicated tests a bit more readable I think.

Reply:
Bonus points(follow-up PR maybe?): Layer1OwnerPawn and Layer1ChildActor, set up the parenting relationship, then add both those clases to the actor list for Layer1.

Reply:
done, except for the bonus points, not sure i understand.